traffic/configure-ios.md

```diff
@@ -15,10 +15,10 @@ This article describes how to use Fiddler Everywhere to capture and inspect traf
 To capture and inspect traffic on iOS devices or simulators, perform the following steps:
 
 1. [Provide the prerequisites](#prerequisites).
-1. [Configure Fiddler Everywhere](#configure-fiddler-everywhere).
-1. [Configure the iOS device](#configure-the-ios-device).
-1. [Configure the iOS simulator](#configure-the-ios-simulator).
-1. [Inspect the traffic](#inspect-the-browser-traffic).
+1. [Configure Fiddler Everywhere](#Configuring-fiddler-everywhere).
```

Comment:
why here it is starting with capital case, while for the rest of the items with lower case?